### ISSUE TITLE
Status of this project

### ISSUE THREAD

**Issue by TheCelavi** [AOP-PHP/AOP#87]
I have searched trough every part of Internet looking for proper support for AOP in PHP. IMHO, you are doing it right, AOP ought to be supported in PHP core, or at least, via extension - no proxy class, no scanning, just intercepting on runtime-level.

Now, please tell me that this project is not dead and your extension is still in development and maintenance :) 

It would be horrible, horrible for this project to die :(

**Comment by TheCelavi** [comment]
And I can not believe that AOP is not part of the PHP core, nor it is proposed at all (at least, I can not find it on official website...)...

**Comment by TheCelavi** [comment]
Hello? Is anyone able to notify us about status of this project? Thanks!

**Comment by geraldcroes** [comment]
Hi. On hollidays right now. Not abandonned though not super active. We have
to manage to save some time to work on the project, time is the main issue
here. We will probably organize the tasks to take this into account.

**Comment by tonix-tuft** [comment]
Yeah, do not abandon this project, it is such a great and useful extension!!!

**Comment by ptx0** [comment]
:+1:

**Comment by ifeltsweet** [comment]
+1
